email_reply_parser/__init__.py

```diff
@@ -63,15 +63,15 @@ class EmailMessage(object):
         r"|(^Skickat fran .*(?:\r?\n(?!\r?\n).*)*)"  # Swedish
         r"|(^Gesendet mit.*(?:\r?\n(?!\r?\n).*)*)"  # German
         r"|(^Hanki .*(?:\r?\n(?!\r?\n).*)*)"  # Finnish
-        r"|(^Lähetetty .*(?:\r?\n(?!\r?\n).*)*)"  # Finnish
         r"|(^Lahetetty .*(?:\r?\n(?!\r?\n).*)*)"  # Finnish
         r"|(^Sendt fra .*(?:\r?\n(?!\r?\n).*)*)"  # Danish
         r"|(^Enviado de .*(?:\r?\n(?!\r?\n).*)*)"  # Portuguese
         r"|(^Enviado desde .*(?:\r?\n(?!\r?\n).*)*)"  # Portuguese
         r"|(^Enviado do .*(?:\r?\n(?!\r?\n).*)*)"  # Portuguese
-        r"|(^Obter o Outlook para Android.*(?:\r?\n(?!\r?\n).*)*)" #Portuguese
+        r"|(^Obter o Outlook para Android.*(?:\r?\n(?!\r?\n).*)*)"  # Portuguese
         r"|(^Verstuurd vanaf .*(?:\r?\n(?!\r?\n).*)*)"  # Dutch
-        r"|(^Envoye .*(?:\r?\n(?!\r?\n).*)*)"  # French
+        r"|(^Envoye de .*(?:\r?\n(?!\r?\n).*)*)"  # French
```

Comment:
I think we should expand this to include a few things like "Envoye depuis mon appareil Galaxy", "Envoye depuis Yahoo Mail pour Android" (these can probably be merged into one), and "Envoye a partir de Outlook pour Android"